SocialFilter/sections/general_elemhide.txt

```diff
@@ -59,6 +65,52 @@
 ##.p-article__articleWrapper ~ div.p-article__share
 ! hide share button in youtube player
 ##.ytp-share-button
+! X (formerly Twitter)
+~twitter.com,~x.com##[title="Compartir en Twitter"]
+~twitter.com,~x.com##body .twitter-share-button
+~twitter.com,~x.com##input.tweet
+~twitter.com,~x.com##[onclick^="window.open('https://x.com/intent/tweet"]
+~twitter.com,~x.com##a[onclick^="window.open('https://x.com/share"]
+~twitter.com,~x.com##[onclick^="window.open('https://twitter.com/intent/tweet"]
+~twitter.com,~x.com##a[onclick^="window.open('https://twitter.com/share"]
+##a[onclick^="window.open('http://x.com/share"]
+##a[onclick^="window.open ('http://www.twitter.com/share?url="]
+~twitter.com,~x.com##a[onclick*="window.open"][onclick*="'https://twitter.com/intent/tweet?"]
+~twitter.com,~x.com##a[onclick^="window.open('http://twitter.com/home?status"]
+##a[onclick^="window.open('http://twitter.com/share"]
+~twitter.com,~x.com##a[href^="https://www.x.com/share"]
+~twitter.com,~x.com##a[href^="https://www.twitter.com/share"]
+~twitter.com,~x.com##a[href^="https://x.com/home/?"]
+~twitter.com,~x.com##a[href^="https://x.com/home?"]
+~twitter.com,~x.com##a[href^="https://x.com/intent/retweet?tweet_id"]
+~twitter.com,~x.com##a[href^="https://twitter.com/home/?"]
+~twitter.com,~x.com##a[href^="https://twitter.com/home?"]
+~twitter.com,~x.com##a[href^="https://twitter.com/intent/retweet?tweet_id"]
+##a[href^="http://www.twitter.com/home?"]
+##a[href^="http://www.twitter.com/share"]
+##a[onclick^="window.open('//x.com/intent/tweet?url="]
+##a[onclick^="window.open('//twitter.com/intent/tweet?url="]
+~twitter.com,~x.com##a[href^="https://x.com/share"]
+~twitter.com,~x.com##a[href^="https://twitter.com.com/share"]
+~twitter.com,~x.com##a[href^="https://www.x.com/intent/tweet?"]
+~twitter.com,~x.com##a[href^="https://www.twitter.com/intent/tweet?"]
+~twitter.com,~x.com##a[href*="http://x.com/share"]
+~twitter.com,~x.com##a[href*="http://twitter.com/share"]
+~twitter.com,~x.com##a[href^="http://x.com/?status="]
+~twitter.com,~x.com##a[href^="http://x.com/home/?"]
+~twitter.com,~x.com##a[href^="http://x.com/home?"]
+~twitter.com,~x.com##a[href^="//x.com/intent/tweet?"]
+~twitter.com,~x.com##a[href^="http://x.com/intent/tweet?"]
+~twitter.com,~x.com##a[href^="http://x.com/intent/tweet/?"]
```

Comment:
Remove them, e.g. `##a[href^="http://x.com/intent/tweet/?"]` has a few publicwww hits but no actual use if you visit the pages. Should I spend time to demonstrate for each rules? Better to remove all added `x.com` rules except for those actually used. There rules most affect performance.